src/core/validators/validators.ts

```diff
@@ -1,6 +1,10 @@
 import * as _ from 'lodash';
 
-type Value = string | { [key: string]: string } | { [key: string]: string }[];
+type Value =
+  | string
+  | { [key: string]: string }
+  | { [key: string]: string }[]
+  | Date;
```

Comment:
This type looks too specific, we cannot really predict what the user will pass as value
It can be string[], number, number[], .... or others

I am thinking the other way around, what do we **not** accept...? only Function?

Reply:
Yes, you are right. We should have to add each time there is a new type of value. Should it be any?